projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -58,8 +58,14 @@ function register_block() {
 		return;
 	}
 
-	if ( ! Jetpack_Gutenberg::is_newsletter_enabled() ) {
-		return; // Stop here if the Newsletter feature is not enabled.
+	/**
+	 * Do not proceed if the newsletter feature is not enabled
+	 */
+	if (
+		/** This filter is documented in class.jetpack-gutenberg.php */
+		! apply_filters( 'jetpack_subscriptions_newsletter_feature_enabled', false )
+	) {
+		return;
```

Comment:
Interesting. So it wasn't saving because it was not registering a meta here?

Reply:
Still trying to figure out what's happening here. I reverted to the previous code as this code wasn't being run when we needed it to